views/history.hbs

```diff
@@ -100,31 +186,44 @@
         </div>
 
         <!-- 2019 -->
-        <div id="nineteen" class="history-year clearfix">
-            <div class="section-header">
-                <span style="background-color: #fafafa; padding: 20px;">
-                    <img class="year-title" src="/img/history/2019 - travel upgrade.png" alt="2019 - Travel Upgrade">
-                </span>
+        <div class="white-tab-container">
+            <div>
+                <div class="section-header">
+                    <h1>2019</h1>
+                </div>
```

Comment:
Like I mentioned above, it's the `section-header` class that's messing up this tab and the other ones